typescript/demo/demo.ts

```diff
@@ -104,14 +105,10 @@ async function createAIConfig() {
   };
 
   const aiConfig = AIConfigRuntime.create("demo", "this is a demo AIConfig");
-  const result = await aiConfig.serialize(model, data, "demoPrompt");
+  const prompts: Prompt[] = await aiConfig.serialize(model, data, "demoPrompt");
```

Comment:
nit, is the explicit typing actually needed here/elsewhere? Should be inferred from the `serialize` response now

Reply:
I like the explicit type because it makes it easier for people to know immediately what type it is